src/coreComponents/mesh/generators/WellGeneratorBase.cpp

```diff
@@ -528,54 +531,53 @@ void WellGeneratorBase::debugWellGeometry() const
     return;
   }
 
-  std::cout << std::endl;
-  std::cout << "++++++++++++++++++++++++++" << std::endl;
-  std::cout << "WellGeneratorBase = " << getName() << std::endl;
-  std::cout << "MPI rank = " << MpiWrapper::commRank( MPI_COMM_GEOSX ) << std::endl << std::endl;
-  std::cout << "Number of well elements = " << m_numElems << std::endl;
+  //1. formatting data
+  TableLayout tableWellLayout = TableLayout( {
+      TableLayout::ColumnParam{"Element no.", TableLayout::Alignment::right},
+      TableLayout::ColumnParam{"CoordX", TableLayout::Alignment::middle},
+      TableLayout::ColumnParam{"CoordY", TableLayout::Alignment::middle},
+      TableLayout::ColumnParam{"CoordZ", TableLayout::Alignment::middle},
+      TableLayout::ColumnParam{"Prev\nElement", TableLayout::Alignment::right},
+      TableLayout::ColumnParam{"Next\nElement", TableLayout::Alignment::right},
+    } );
+
+  tableWellLayout.setTitle( "InternalWellGenerator " + getName());
+
+  TableData tableWellData;
 
+  //2. collecting data
   for( globalIndex iwelem = 0; iwelem < m_numElems; ++iwelem )
   {
-    std::cout << "Well element #" << iwelem << std::endl;
-    std::cout << "Coordinates of the element center: " << m_elemCenterCoords[iwelem] << std::endl;
-    if( m_nextElemId[iwelem] < 0 )
-    {
-      std::cout << "No next well element" << std::endl;
-    }
-    else
-    {
-      std::cout << "Next well element # = " << m_nextElemId[iwelem] << std::endl;
-    }
-    if( m_prevElemId[iwelem][0] < 0 )
-    {
-      std::cout << "No previous well element" << std::endl;
-    }
-    else
+    std::optional< globalIndex > nextElement;
+    std::optional< globalIndex > prevElement;
+
+    if( m_nextElemId[iwelem] >= 0 )
     {
-      std::cout << "Previous well element #" << m_prevElemId[iwelem][0] << std::endl;
+      nextElement =  m_nextElemId[iwelem];
     }
-    for( globalIndex inode = 0; inode < m_numNodesPerElem; ++inode )
+
+    if( m_prevElemId[iwelem][0] >= 0 )
     {
-      if( inode == 0 )
-      {
-        std::cout << "First well node: #" << m_elemToNodesMap[iwelem][inode] << std::endl;
-      }
-      else
-      {
-        std::cout << "Second well node: #" << m_elemToNodesMap[iwelem][inode] << std::endl;
-      }
+      prevElement =  m_prevElemId[iwelem][0];
     }
+
+    tableWellData.addRow( iwelem, m_elemCenterCoords[iwelem][0], m_elemCenterCoords[iwelem][1], m_elemCenterCoords[iwelem][2], prevElement, nextElement );
```

Comment:
long long line
```suggestion
    tableWellData.addRow( iwelem,
                          m_elemCenterCoords[iwelem][0], m_elemCenterCoords[iwelem][1], m_elemCenterCoords[iwelem][2],
                          prevElement, nextElement );
```
or
```suggestion
    tableWellData.addRow( iwelem,
                          m_elemCenterCoords[iwelem][0],
                          m_elemCenterCoords[iwelem][1],
                          m_elemCenterCoords[iwelem][2],
                          prevElement,
                          nextElement );
```